cli/lib/kontena/cli/master/users_command.rb

```diff
@@ -1,6 +1,4 @@
 module Kontena::Cli::Master
-
-
   class UsersCommand < Kontena::Command
```

Comment:
Seems like the model of CLI consistency is to rename this to `cli/master/user_command.rb` `class UserCommand`, but leave the `cli/master/users/*_command.rb` as-is.